### ISSUE TITLE
it seems to be wrong progbar when verbose = 1 in the model.fit function

### ISSUE THREAD

**Issue by cyzhangAThit** [fchollet/keras#4123]
Hi,
    I am a keras rookie. I have installed the keras and theano successfully at the latest version with the following command :
1. pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps
2. pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps 
   
   And I run some toy code, and got a strange output like that.

from keras.models import Sequential
from keras.layers import Dense
import numpy
seed = 7
numpy.random.seed(seed)
X = numpy.random.random((768,8))
Y = numpy.random.randint(2,size=(768,1))
model = Sequential()
model.add(Dense(12, input_dim=8, init='uniform', activation='relu'))
model.add(Dense(8, init='uniform', activation='relu'))
model.add(Dense(1, init='uniform', activation='sigmoid'))
model.compile(loss='binary_crossentropy', optimizer='adam', metrics=['accuracy'])
model.fit(X, Y, nb_epoch=150, batch_size=10, verbose=1)
scores = model.evaluate(X, Y, batch_size=40,verbose=1)
print("%s: %.2f%%" % (model.metrics_names[1], scores[1]*100))

the output is as follows:
768/768 [==============================] - 0s - loss: 0.6738 - acc: 0.5846
Epoch 147/150
768/768 [==============================] - 0s - loss: 0.6742 - acc: 0.5885
Epoch 148/150
768/768 [==============================] - 0s - loss: 0.6728 - acc: 0.5846
Epoch 149/150
768/768 [==============================] - 0s - loss: 0.6741 - acc: 0.5859
Epoch 150/150
768/768 [==============================] - 0s - loss: 0.6750 - acc: 0.5859
 40/768 [>.............................] - ETA: 0sacc: 59.37%

The progbar just run 1 batch and breaked.
After some limited debugging, I think it' wrong with the model.fit when verbose is set to 1.
So what I should do to solve it? It's seem to be OK just setting verbose to 0.

**Comment by diegow88** [comment]
Hi @cyzhangAThit, did you figure what was the reason? I have the same problem. It just stops after certain number of batches.
Thanks
